stacks/serverless/dovetail-cdn-arranger.yml

```diff
@@ -0,0 +1,133 @@
+# stacks/serverless/dovetail-cdn-arranger.yml
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Dovetail CDN Arranger lambda function
+Conditions:
+  IsProduction: !Equals [!Ref EnvironmentType, Production]
+Parameters:
+  CodeS3Bucket:
+    Type: String
+  CodeS3ObjectVersion:
+    Type: String
+  EnvironmentType:
+    Type: String
+  DovetailHost:
+    Type: AWS::SSM::Parameter::Value<String>
+  DovetailToken:
+    Type: AWS::SSM::Parameter::Value<String>
+  DynamodbTableArn:
+    Type: AWS::SSM::Parameter::Value<String>
+Resources:
+  DovetailCdnArrangerWorkspaceBucket:
```

Comment:
Because of how S3 bucket names are derived, this will almost certainly be truncated to something like `………-DovetailC`. It's useful to front load the resource logical name in this case with the most descriptive part, to make it easier to identify and find the bucket, even though it does break with resource naming conventions.